monitors.yml

```diff
@@ -123,9 +123,6 @@ monitors:
     options:
       notify_no_data: true
       require_full_window: true
-      thresholds:
-        critical: 1.0
-        warning: 1.0
```

Comment:
So this was only changed recently, maybe the incompatible bit is warning == critical? It was working before and i'm pretty sure we do need these.